### ISSUE TITLE
Mock server returns int64 value less than the lower limit

### ISSUE THREAD

**Issue by atken** [stoplightio/prism#1992]
## Describe the bug

The range of int64 is -9223372036854775808 (-2^63) ~ 9223372036854775807 (2^63-1), but the mock server seems to return -9223372036854776000 by default.

## To Reproduce

1. Install prism.

```
npm install -g @stoplight/prism-cli
```

2. Start a mock server with the following command.

```
prism mock https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore-expanded.yaml
```

3. Call a REST API that returns a response containing an int64 value.

```
curl -X GET "http://localhost:4010/pets/1"
{"name":"string","tag":"string","id":-9223372036854776000}
```

## Expected behavior

The mock server returns -9223372036854775808 or a value in the range of int64.

## Additional context

**Environment (remove any that are not applicable):**
 - Library version: 4.7.0
 - OS: Pop!_OS 21.04
 - Browser: Not using
 - Platform: Desktop

**Comment by Riverlution** [comment]
I've just encountered this, any info on when we might expect a fix?

**Comment by daniel-white** [comment]
Although JSON and JSON Schema appears to allow int64 as a literal numeric value, Prism is written to run NodeJS. JavaScript has an inherit limitation on the size of the `number` capping the effective limit to 56 bits for integers. There for it would be a challenge for Prism to properly set values outside of this range.

There are discussions in OpenAPI and [JSON Schema](https://github.com/json-schema-org/json-schema-spec/issues/574) as how to best handle and/or document this.

**Comment by jedwards1211** [comment]
@daniel-white there are tools like Long.js (and of course now there is BigInt) that make it easy enough to work around the limitations of JS numbers.
But if you're not going to use something like that can you at least make sure Prism generates a valid response (some number within int64 range)?  We're getting test errors in a Java SDK because the response from prism is invalid.

**Comment by rattrayalex** [comment]
I think the ask here is as simple as a default return value for `int64` of `-9223372036854775808` instead of `-9223372036854776000`, which I assume should be a straightforward fix and easy to accept? (I haven't taken a look at the code).

@ilanashapiro could you please take a crack at this?

**Comment by rattrayalex** [comment]
https://github.com/stoplightio/http-spec/pull/253 should fix this if merged

**Comment by chohmann** [comment]
This was fixed in version 5.5.1.
